src/components/select/select.spec.js

```diff
@@ -153,6 +153,16 @@ describe('<md-select>', function() {
     expect($rootScope.myForm.select.$touched).toBe(true);
   }));
 
+  it('should not set touched if the element is disabled', inject(function($compile, $rootScope) {
```

Comment:
Good point, this one is useless after the latest changes. It probably makes more sense to test whether the tabindex gets removed.